README.md

```diff
@@ -1,3 +1,54 @@
 # data-lake-alerts
 
-Query the data lake and alert if thresholds are breached
\ No newline at end of file
+Allows developers to detect production issues, by automating the process of querying the Data Lake 
```

Comment:
Probably not important to change now, but developers could easily add alerting for positive events rather than just undesirable results.

Reply:
Yes, that's a good point, I am a natural pessimist so I'd never have thought to do that 😄